packages/pagination/src/ux-pagination.ts

```diff
@@ -0,0 +1,83 @@
+import { customElement, useView, PLATFORM, bindingMode } from 'aurelia-framework';
+import { bindable } from 'aurelia-typed-observable-plugin';
```

Comment:
@EisenbergEffect is it ok to go ahead?

Reply:
I would prefer to resolve the design issue with au2 and then bake the same design into au1 core.

Reply:
@bigopon What needs to be done to progress this?

Reply:
Need to settle on how we would want to API to be for type declaration shortcut:
```ts
@bindable({ mode: 'toggle' })
@bindable({ type: 'bool' })
@bindable({ coerce: 'bool' })
@bindable({ set: 'bool' })
```

Reply:
I think for now, we can settle first with `set` api supporting only a setter function, as it's already in v2. For more high level api like `type/mode/coerce`, we can enhance it later. Does that sound good to start adding it to v1?

Reply:
@EisenbergEffect any comment on the above ☝️